docs/ABOUT.md

```diff
@@ -20,28 +20,12 @@ Python can be applied to many different classes of problems [what python is good
 - [Network Programming](https://www.python.org/about/apps/#network-programming)
 - [Software & Game Development](https://www.python.org/about/apps/#software-development)
 
-### Who uses python?
-
-Let the names of just some companies talk by themselves[Success](https://www.python.org/about/success/).
-
-- [Industrial Light & Magic Runs on Python](https://www.python.org/about/success/ilm/)
-- [D-Link Australia Uses Python to Control Firmware Updates](https://www.python.org/about/success/dlink).
-- [ForecastWatch.com Uses Python To Help Meteorologists](https://www.python.org/about/success/forecastwatch/).
-- [Frequentis TAPtools® - Python in Air Traffic Control](https://www.python.org/about/success/frequentis/).
-- [Maritime Industry Increases Efficiency with Python](https://www.python.org/about/success/tribon/).
-- [Python Streamlines Space Shuttle Mission Design](https://www.python.org/about/success/usa/).
-- [Python is Rackspace's CORE Technology](https://www.python.org/about/success/rackspace/).
-- [ERP5: Mission-critical ERP/CRM with Python and Zope](https://www.python.org/about/success/nexedi/).
-- Many others: Google, NASA, Intel, AMD, Walt Disney Animation Studios, Canonical, Red Hat, Amazon, Netflix, and thousands upon thousands more.
-
 ### Reasons to learn
 
 Some reasons to learn Python [Reasons to learn](http://www.skilledup.com/articles/reasons-to-learn-python):
 
 - It is easy to learn. Python was designed to help the developer not the machine.
 - It helps the novice programmer. Concepts used in python can be applied to other programming languages.
 - It is fun. Some projects like [Raspberry Pi](http://www.raspberrypi.org/faqs#introWhatIs) uses Python as the main language and can be used to create robots, remote controlled cars and video game consoles.
-- There are many job opportunities for Python developers in companies like Google, Yahoo!, Disney, Nokia, and IBM. Not to mention a huge amount of startups. 
-- It have a wide range of open source web application frameworks.
 
-Python official web site [python website](https://www.python.org) and [python documentation](https://docs.python.org/) documentation to learn more.
+There are many job opportunities for Python developers in companies like Google, Yahoo!, Disney, Nokia, and IBM. Not to mention a huge amount of startups. It have a wide range of open source web application frameworks. Visit the Python official web site [python website](https://www.python.org) and [python documentation](https://docs.python.org/) documentation to learn more.
```

Comment:
I would use "number of" instead of "amount of" when talking about startups, since startups are countable rather than measurable.